ouroboros-consensus/src/Ouroboros/Consensus/ChainSyncClient.hs

```diff
@@ -258,7 +282,7 @@ chainSyncClient tracer cfg btime (ClockSkew maxSkew) getCurrentChain
         { candidateChain      = candidateChain'
         , candidateChainState = candidateChainState'
         }
-      return $ requestNext varCandidate
+      return $ requestNext
```

Comment:
HLint is going to complain that you have a redundant `$` (you did remove it on L319, L420, and L461, though).

Reply:
done